Tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -355,4 +355,24 @@ public function testProfilingToolbarCollision()
         $file = __DIR__.'/../../Resources/Fixtures/config/bc/profiling_toolbar.yml';
         $this->assertProcessedConfigurationEquals([], [$file]);
     }
+
+    /**
+     * @expectedException \Symfony\Component\Config\Definition\Exception\InvalidConfigurationException
+     * @expectedExceptionMessage The child node "cache_pool" at path "httplug.clients.test.plugins.0.cache" must be configured.
```

Comment:
please use $this->expectException and $this->expectExceptionMessage instead of the annotations. the annotations have been deprecated in favor of the method. they have the upsides of being able to use the ::class constant and (not relevant in this case here) of only accepting the exception at the place you actually expect it, not before that.

Reply:
There is no instance in this code base where expectExceptionMessage is used. I've followed style of existing code. 

Furthermore, expectException is PHPUnit 6+ only.

If you decided to change the code style, please do it yourself with refactoring commit which replaces it in all places at once.

Reply:
sorry, my bad. i am working in too many repositories that are at different states of upgrading those things... indeed, its better to be consistent with this. i created #248